controllers/bootstrap/init.go

```diff
@@ -813,7 +819,40 @@ func (b *Bootstrap) renderTemplate(objectTemplate string, data interface{}, alwa
 // 	return ""
 // }
 
-func (b *Bootstrap) updateOperandRegistry(installPlanApproval olmv1alpha1.Approval) error {
+func (b *Bootstrap) UpdateCsOpApproval() error {
+	sub := &olmv1alpha1.Subscription{}
+	subKey := types.NamespacedName{
+		Name:      "ibm-common-service-operator",
```

Comment:
I noticed that we define `IBMCSPackage = "ibm-common-service-operator"` in `constant.go`
Maybe we should this variable to replace the string here?

Reply:
This one should be fine, it is the name of the subscription. I will replace CS packagenames with IBMCSPackage in the following comments.